sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/query.sql

```diff
@@ -1,18 +1,74 @@
-WITH _events_ping_distinct_client_count AS (
+WITH baseline_clients AS (
   SELECT
-    DATE(submission_timestamp) AS submission_date,
-    COUNT(DISTINCT client_info.client_id) AS events_ping_distinct_client_count
+    DATE(
+      DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC')
+    ) AS ping_date,
+    client_info.client_id,
+    normalized_channel AS channel,
+    normalized_country_code AS country
   FROM
-    firefox_ios.events_unnested
+    firefox_ios.baseline
   WHERE
-    DATE(submission_timestamp) = @submission_date
+    metrics.timespan.glean_baseline_duration.value > 0
+    AND LOWER(metadata.isp.name) <> "browserstack"
+    AND DATE(submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 4 DAY)
+    AND @submission_date
+    AND DATE(
+      DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC')
+    ) = DATE_SUB(@submission_date, INTERVAL 4 DAY)
+  QUALIFY
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        client_info.client_id
+      ORDER BY
+        DATE(DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC'))
+    ) = 1
+),
+client_attribution AS (
+  SELECT
+    client_id,
+    channel,
+    adjust_network,
+  FROM
+    firefox_ios.firefox_ios_clients
+),
+default_browser AS (
```

Comment:
My guess is that we could end up with multiple records for a single user here? We should probably make sure that does not happen to avoid accidental duplication downstream.

Reply:
@kik-kik only 391 users out of >2 million have multiple default browser records. I added a line to include only one record for each user.

Reply:
Thanks!